packages/bot/src/transformers/member.ts

```diff
@@ -35,6 +35,7 @@ export function transformMember(bot: Bot, payload: DiscordMember, guildId: BigSt
   if (props.deaf || props.mute || props.pending) {
     member.toggles = new MemberToggles(payload)
   }
+  if (props.flags && payload.flags) member.flags = payload.flags
```

Comment:
```suggestion
  if (props.flags && 'flags' in payload) member.flags = payload.flags
```
I'm pretty sure flags can be 0?

Reply:
Oh right, i need to add !== undefined then, (see my comment on the invite type pr, i'm not going to use `in` in this case)